packages/runed/src/lib/utilities/index.ts

```diff
@@ -26,3 +26,4 @@ export * from "./use-resize-observer/index.js";
 export * from "./watch/index.js";
 
 export * from "./scroll-state/index.js";
+export * from "./use-search-params/index.js";
```

Comment:
I don't think we can do this...we are importing sveltekit stuff in the module that will fail if you try to use runed in a svelte only project.

Reply:
yeah, we should have a /kit sub-import path `import * from 'runed/kit'`